admin/src/scenes/plan-transport/ligne-bus/List.jsx

```diff
@@ -196,17 +203,21 @@ const ReactiveList = ({ cohort, history }) => {
             />
           </div>
           <div className="flex items-center gap-2">
-            <button
-              className="text-grey-700 flex h-10 items-center gap-2 rounded-md border border-gray-300 bg-white px-3 text-sm font-medium"
-              onClick={() => history.push(`/ligne-de-bus/historique?cohort=${cohort}`)}>
-              <History className="text-gray-400" />
-              Historique
-            </button>
-            <button
-              className="h-10 rounded-md border border-gray-300 bg-white px-3 text-sm font-medium text-gray-700"
-              onClick={() => history.push(`/ligne-de-bus/demande-de-modification?cohort=${cohort}`)}>
-              Demande de modification
-            </button>
+            {user.role !== ROLES.HEAD_CENTER ? (
+              <button
+                className="text-grey-700 flex h-10 items-center gap-2 rounded-md border border-gray-300 bg-white px-3 text-sm font-medium"
+                onClick={() => history.push(`/ligne-de-bus/historique?cohort=${cohort}`)}>
+                <History className="text-gray-400" />
+                Historique
+              </button>
+            ) : null}
+            {user.role !== ROLES.HEAD_CENTER ? (
+              <button
+                className="h-10 rounded-md border border-gray-300 bg-white px-3 text-sm font-medium text-gray-700"
+                onClick={() => history.push(`/ligne-de-bus/demande-de-modification?cohort=${cohort}`)}>
+                Demande de modification
+              </button>
+            ) : null}
```

Comment:
On peut faire une seule condition

```suggestion
            {user.role !== ROLES.HEAD_CENTER ? (
              <>
                <button
                  className="text-grey-700 flex h-10 items-center gap-2 rounded-md border border-gray-300 bg-white px-3 text-sm font-medium"
                  onClick={() => history.push(`/ligne-de-bus/historique?cohort=${cohort}`)}>
                  <History className="text-gray-400" />
                  Historique
                </button>
                <button
                  className="h-10 rounded-md border border-gray-300 bg-white px-3 text-sm font-medium text-gray-700"
                  onClick={() => history.push(`/ligne-de-bus/demande-de-modification?cohort=${cohort}`)}>
                  Demande de modification
                </button>
              </>
            ) : null}
```